Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -29,10 +30,31 @@ export function findReferencingDocumentIdsSql(referencedDocumentIds: string[]):
  * @param documentId the id of the document to find aliases for
  * @returns a SQL query to find the alias ids of the given document
  */
-export function findAliasIdsForDocumentSql(documentId: string): string {
+export function findAliasIdsForDocumentSql(documentId: MeadowlarkId): string {
```

Comment:
Can we change `documentId` everywhere to the actual type of id to avoid any confusion? For example, naming it `meadowlarkId` here would be more appropriate.